src/Symfony/Component/HttpKernel/DataCollector/DumpDataCollector.php

```diff
@@ -140,6 +174,7 @@ public function serialize()
         $this->data = array();
         $this->dataCount = 0;
         $this->isCollected = true;
+        $this->dumper = null;
```

Comment:
I just added this new line to make the service reusable, aka state less after use